lfs/gitscanner_remotes.go

```diff
@@ -13,7 +13,7 @@ import (
 // remote, use a more explicit diff command.
 func calcSkippedRefs(remote string) []string {
 	cachedRemoteRefs, _ := git.CachedRemoteRefs(remote)
-	actualRemoteRefs, _ := git.RemoteRefs(remote)
+	actualRemoteRefs, _ := git.RemoteRefs(remote, true)
 
 	// The set of remote refs can be very large. Since CachedRemoteRefs only
 	// returns branches, filter the remote refs and convert them to a set for
```

Comment:
I notice that we can also simplify the following loop because we no longer need to filter for branch refs only.  I've written up that simplification and can push a commit to this branch, if you're willing to accept it.  Otherwise I'll put it into a separate branch in my fork, and you can take a look at it there.

Reply:
Hey Chris, please feel free to add any commits to this.